### PR TITLE
Added mobile and desktop version of website

### PR DESCRIPTION
**Pull request by FlameNV**
Виконав перше домашнє завдання

### CODE REVIEW

File: AltexSoft lab 1/index.html [hunk 1]
```diff
@@ -0,0 +1,773 @@
...
+        </div>
+        <h1 class="header-title">TIPS & TOOLS</h1>
+      </header>
+      <div class="navbar">
+        <a href="#tips" class="navbar-link">tips</a>
+        <a href="#glossary" class="navbar-link">glossary</a>
+        <a href="#faqs" class="navbar-link">faqs</a>
+      </div>
+      <main class="main-section">
+        <a name="tips"></a>
```
Comment (marynakorol) [inline AltexSoft lab 1/index.html]: missed href attr here

File: AltexSoft lab 1/index.html [hunk 2]
```diff
@@ -0,0 +1,773 @@
...
+            </ul>
+            <button class="menu-get-skout-btn" id="get-skout-btn">
+              get skout
+            </button>
+            <button class="menu-login-btn" id="login-btn">log in</button>
+          </div>
+        </div>
+        <h1 class="header-title">TIPS & TOOLS</h1>
+      </header>
+      <div class="navbar">
```
Comment (marynakorol) [inline AltexSoft lab 1/index.html]: you can use nav tag here

File: AltexSoft lab 1/index.html [hunk 3]
```diff
@@ -0,0 +1,773 @@
...
+            class="logo-dark"
+            id="logo-dark"
+          />
+          <button class="menu-btn" onclick="openMenu()">
+            <svg class="icon menu-icon" id="menu-icon">
+              <use xlink:href="./images/icons.svg#menu"></use>
+            </svg>
+          </button>
+
+          <div id="nav-links" class="nav-links">
```
Comment (marynakorol) [inline AltexSoft lab 1/index.html]: use nav

File: AltexSoft lab 1/index.html [hunk 4]
```diff
@@ -0,0 +1,773 @@
...
+        <section class="skout-tips">
+          <img
+            src="images/skout-tips.svg"
+            alt="skout tips"
+            class="skout-tips-img"
+          />
+          <div class="accordion-wrapper">
+            <span class="accordion-delimiter"></span>
+            <button class="accordion-btn">
+              <p class="accordion-number">1</p>
```
Comment (marynakorol) [inline AltexSoft lab 1/index.html]: you can not use  p inside button change this please

File: AltexSoft lab 1/index.html [hunk 5]
```diff
@@ -0,0 +1,773 @@
...
+                  to reach target and hackers will often go after the lowest
+                  hanging fruit. When it comes to maintaining a strong
+                  cybersecurity network the best thing a user can do is to
+                  prevent problems before they occur, and the easiest way to do
+                  that is by making sure that all systems are updated and
+                  updated regularly.
+                </p>
+                <div class="accordion-note-block">
+                  <span class="note-icon">
+                    <p class="note-icon-letter">i</p>
```
Comment (marynakorol) [inline AltexSoft lab 1/index.html]: p can not be a child of span please reverse them

File: AltexSoft lab 1/index.html [hunk 6]
```diff
@@ -0,0 +1,773 @@
...
+                  </span>
+                  <p class="accordion-note">
+                    Based on the 2018 Verizon Data Breach Report
+                  </p>
+                </div>
+              </div>
+            </div>
+            <span class="accordion-delimiter"></span>
+            <button class="accordion-btn">
+              <p class="accordion-number">2</p>
```
Comment (marynakorol) [inline AltexSoft lab 1/index.html]: the same, wrong semantically

File: AltexSoft lab 1/index.html [hunk 7]
```diff
@@ -0,0 +1,773 @@
...
+                  servers, laptops, or any other kind of device in the IoT.
+                  Malware often takes the form of code, innocuous content,
+                  scripts or software. With SKOUT’s End-to-End Protection you
+                  can rest assured that all vulnerable points within your system
+                  are accounted for and safeguarded so that you don’t fall
+                  victim to malware of any kind.
+                </p>
+                <div class="accordion-note-block">
+                  <span class="note-icon">
+                    <p class="note-icon-letter">i</p>
```
Comment (marynakorol) [inline AltexSoft lab 1/index.html]: reverse span and p

File: AltexSoft lab 1/index.html [hunk 8]
```diff
@@ -0,0 +1,773 @@
...
+                  </span>
+                  <p class="accordion-note">
+                    Based on the 2018 Verizon Data Breach Report
+                  </p>
+                </div>
+              </div>
+            </div>
+            <span class="accordion-delimiter"></span>
+            <button class="accordion-btn">
+              <p class="accordion-number">3</p>
```
Comment (marynakorol) [inline AltexSoft lab 1/index.html]: do not use p inside a button

File: AltexSoft lab 1/index.html [hunk 9]
```diff
@@ -0,0 +1,773 @@
...
+                  cybersecurity requirements are met. Requirements could range
+                  from individuals setting up approved firewalls to everyone
+                  within a team taking on a complex password system. Small
+                  changes often make all the difference when trying to avoid a
+                  cybersecurity crisis and will allow you to find trouble before
+                  trouble finds you.
+                </p>
+                <div class="accordion-note-block">
+                  <span class="note-icon">
+                    <p class="note-icon-letter">i</p>
```
Comment (marynakorol) [inline AltexSoft lab 1/index.html]: GLOBAL: do not use block elements inside inline ones

File: AltexSoft lab 1/index.html [hunk 10]
```diff
@@ -0,0 +1,773 @@
...
+      </header>
+      <div class="navbar">
+        <a href="#tips" class="navbar-link">tips</a>
+        <a href="#glossary" class="navbar-link">glossary</a>
+        <a href="#faqs" class="navbar-link">faqs</a>
+      </div>
+      <main class="main-section">
+        <a name="tips"></a>
+        <div class="section-delimiter"></div>
+        <section class="skout-tips">
```
Comment (marynakorol) [inline AltexSoft lab 1/index.html]: you use section tag here, but it doesn't have any heading tag (h2-h6), section should have

File: AltexSoft lab 1/index.html [hunk 11]
```diff
@@ -0,0 +1,773 @@
...
+                  <p class="accordion-note">
+                    Based on the 2018 Verizon Data Breach Report
+                  </p>
+                </div>
+              </div>
+            </div>
+            <span class="accordion-delimiter"></span>
+          </div>
+        </section>
+        <a name="glossary"></a>
```
Comment (marynakorol) [inline AltexSoft lab 1/index.html]: href expected

Comment (marynakorol) [reply AltexSoft lab 1/index.html]: It seems, you need id here

File: AltexSoft lab 1/index.html [hunk 12]
```diff
@@ -0,0 +1,773 @@
...
+                  <span
+                    class="skout-glossary-slider-menu-line"
+                    onclick="currentSlide(21)"
+                  ></span>
+                  <p class="skout-glossary-slider-menu-title" onclick="currentSlide(21)">Worm</p>
+                </div>
+              </div>
+              <div class="skout-glossary-buttons" id="list_index">
+                <span
+                  class="skout-glossary-slider-next-btn"><div class="arrow right"></div
```
Comment (marynakorol) [inline AltexSoft lab 1/index.html]: you can't use div inside a span

File: AltexSoft lab 1/index.html [hunk 13]
```diff
@@ -0,0 +1,773 @@
...
+                ></span>
+                <span
+                  class="skout-glossary-slider-prev-btn"><div class="arrow left"></div
```
Comment (marynakorol) [inline AltexSoft lab 1/index.html]: same

File: AltexSoft lab 1/index.html [hunk 14]
```diff
@@ -0,0 +1,773 @@
...
+                ></span>
+              </div>
+            </div>
+            <div class="skout-glossary-slider-content">
+              <div class="skout-glossary-slider-item">
+                <h1 class="skout-glossary-slider-item-title">Botnet</h1>
```
Comment (marynakorol) [inline AltexSoft lab 1/index.html]: you may have just one h1 on the page

File: AltexSoft lab 1/index.html [hunk 15]
```diff
@@ -0,0 +1,773 @@
...
+        <div class="section-delimiter margin"></div>
+        <section class="skout-faqs">
+          <img
+            src="./images/skout-faqs.svg"
+            alt="skout faqs image"
+            class="skout-faqs-img"
+          />
+          <div class="skout-faqs-accordion">
+            <button class="accordion-btn skout-faqs-accordion-btn">
+              <p class="skout-faqs-q-letter">Q</p>
```
Comment (marynakorol) [inline AltexSoft lab 1/index.html]: should be pseudo-element ::before here to keep DRY principles

File: AltexSoft lab 1/index.html [hunk 16]
```diff
@@ -0,0 +1,773 @@
...
+            <div class="footer-nav-block">
+              <div class="links">
+                <a href="#" class="link">blog</a>
+                <a href="#" class="link">company</a>
+                <a href="#" class="link">terms & conditions</a>
+                <a href="#" class="link">privacy policy</a>
+                <a href="#" class="link">careers</a>
+              </div>
+              <p class="call-the-soc">call the soc</p>
+              <p class="phone-number">1-855-838-4500</p>
```
Comment (marynakorol) [inline AltexSoft lab 1/index.html]: should be a link with tel:

File: AltexSoft lab 1/index.html [hunk 17]
```diff
@@ -0,0 +1,773 @@
...
+                <p class="footer-address-street-info">
+                  423 W 55th St. 7th Floor<br />
+                  New York, NY, 10019
+                </p>
+                <a href="" class="footer-address-map-link">View Map</a>
+              </div>
+              <div class="footer-address-block">
+                <p class="footer-address-info">Corporate Headquarters</p>
+                <p class="phone-number">+1 631-203-6600</p>
+                <p class="footer-address-street-info">
```
Comment (marynakorol) [inline AltexSoft lab 1/index.html]: can be just footer-address-info

File: AltexSoft lab 1/index.html [hunk 18]
```diff
@@ -0,0 +1,773 @@
...
+            </div>
+            <div class="footer-address-block-wrapper">
+              <div class="footer-address-block">
+                <p class="footer-address-info">New York</p>
+                <p class="phone-number">+1 631-203-6600</p>
+                <p class="footer-address-street-info">
+                  423 W 55th St. 7th Floor<br />
+                  New York, NY, 10019
+                </p>
+                <a href="" class="footer-address-map-link">View Map</a>
```
Comment (marynakorol) [inline AltexSoft lab 1/index.html]: simplify class to footer-map-link

File: AltexSoft lab 1/index.html [hunk 19]
```diff
@@ -0,0 +1,773 @@
...
+            </div>
+            <button class="accordion-btn skout-faqs-accordion-btn">
+              <p class="skout-faqs-q-letter">Q</p>
+              <p class="skout-faqs-question">
+                Do I need an MSP or Technology provider to get SKOUT?
+              </p>
+            </button>
+            <div class="skout-faqs-answer">
+              <div class="skout-faqs-answer-wrapper">
+                <p class="skout-faqs-a-letter">A</p>
```
Comment (marynakorol) [inline AltexSoft lab 1/index.html]: move to ::before el

File: AltexSoft lab 1/scss/_media-query.scss [hunk 20]
```diff
@@ -0,0 +1,337 @@
...
+      padding-top: 182px;
+      font-size: 110px;
+      line-height: 130px;
+      width: 50%;
+    }
+
+    .header-nav {
+      padding: 28px 84px 28px 90px;
+
+      .nav-links {
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/_media-query.scss:25]: do not limit a width of links 
image

Comment (marynakorol) [reply AltexSoft lab 1/scss/_media-query.scss:25]: logo is not vertical centered

File: AltexSoft lab 1/index.html [hunk 21]
```diff
@@ -0,0 +1,773 @@
...
+    <link
+      href="https://fonts.googleapis.com/css2?family=Aclonica&family=Acme&family=Actor&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <div class="container">
+      <header class="header">
+        <div class="header-nav" id="navigation">
+          <img src="images/skout-3.png" alt="logo" class="logo" id="logo" />
```
Comment (marynakorol) [inline AltexSoft lab 1/index.html]: should be  a link

File: AltexSoft lab 1/js/hamburger.js [hunk 22]
```diff
@@ -0,0 +1,58 @@
+const menu = document.querySelector("#menu");
+
+function openMenu() {
```
Comment (marynakorol) [inline AltexSoft lab 1/js/hamburger.js:6]: doesn't work correctly on mobile, the button doesn't change its state to close btn, the menu doesn't close after a link is clicked

File: AltexSoft lab 1/js/hamburger.js [hunk 23]
```diff
@@ -0,0 +1,58 @@
...
+      logo.style.display = "block";
+      logoDark.style.display = "none";
+      menuIcon.style.fill = "#fff";
+    } else {
+      logo.style.display = "none";
+      logoDark.style.display = "block";
+      menuIcon.style.fill = "#7F7F7F";
+    }
+  } else {
+    if (this.scrollY <= 450) {
```
Comment (marynakorol) [inline AltexSoft lab 1/js/hamburger.js]: leave a comment where you get this breakpoint or use match media

File: AltexSoft lab 1/js/hamburger.js [hunk 24]
```diff
@@ -0,0 +1,58 @@
...
+      logo.style.display = "block";
+      logoDark.style.display = "none";
+      menu.style.color = "#fff";
+      getSkoutBtn.style.color = "#fff";
+      getSkoutBtn.style.borderColor = "#fff";
+      loginBtn.style.color = "#fff";
+      loginBtn.style.borderColor = "#fff";
+    } else {
```
Comment (marynakorol) [inline AltexSoft lab 1/js/hamburger.js:38]: you need to keep these styles into a specific class in css

File: AltexSoft lab 1/js/hamburger.js [hunk 25]
```diff
@@ -0,0 +1,58 @@
...
+
+window.onscroll = () => {
+  const logo = document.querySelector("#logo");
+  const logoDark = document.querySelector("#logo-dark");
+  const menuIcon = document.querySelector("#menu-icon");
+  const getSkoutBtn = document.querySelector("#get-skout-btn");
+  const loginBtn = document.querySelector("#login-btn");
+
+  if (window.innerWidth < 1440) {
+    if (this.scrollY <= 190) {
```
Comment (marynakorol) [inline AltexSoft lab 1/js/hamburger.js]: hardcode 190, change please

File: AltexSoft lab 1/js/pagination.js [hunk 26]
```diff
@@ -0,0 +1,25 @@
...
+      paginator({
+        get_rows: function () {
+          return document.getElementById("list").getElementsByTagName("div");
+        },
+        box: document.getElementById("list_index"),
+        active_class: "color_page",
+        rows_per_page: 5,
+      });
+    }
+  },
```
Comment (marynakorol) [inline AltexSoft lab 1/js/pagination.js]: image

File: AltexSoft lab 1/scss/_media-query.scss [hunk 27]
```diff
@@ -0,0 +1,337 @@
+@import "variables";
+
+@media only screen and (min-width: 1440px) {
+  .header {
+    height: 500px;
+
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/_media-query.scss]: GLOBAL: all repeatable values should be in vars

File: AltexSoft lab 1/scss/_media-query.scss [hunk 28]
```diff
@@ -0,0 +1,337 @@
...
+          display: inline-block;
+          color: $white-color;
+          .menu-item {
+            margin-bottom: 0px;
+            display: inline-block;
+            .nav-link {
+              color: inherit;
+              margin-right: 50px;
+              padding-bottom: 7px;
+
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/_media-query.scss:60]: GLOBAL: 
- keep box-model hierarchy : position>display>size>indents>text styles>other
- all duplicates should be separated into mixins or helper classes

File: AltexSoft lab 1/scss/_media-query.scss [hunk 29]
```diff
@@ -0,0 +1,337 @@
...
+              &.active {
+                border-bottom: 1.5px solid $white-color;
+              }
+
+              &:hover {
+                color: $color-for-links-and-buttons;
+                transition: .7s;
+              }
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/_media-query.scss:68]: here hover should have underline as active el

File: AltexSoft lab 1/scss/_media-query.scss [hunk 30]
```diff
@@ -0,0 +1,337 @@
...
+      padding: 28px 84px 28px 90px;
+
+      .nav-links {
+        display: block;
+        position: static;
+        height: auto;
+        background-color: transparent;
+        text-align: right;
+
+        .menu {
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/_media-query.scss:32]: please add white bg to header on scrolling

File: AltexSoft lab 1/index.html [hunk 31]
```diff
@@ -0,0 +1,773 @@
...
+          <img
+            src="./images/skout-2.png"
+            alt="logo dark"
+            class="logo-dark"
+            id="logo-dark"
+          />
+          <button class="menu-btn" onclick="openMenu()">
+            <svg class="icon menu-icon" id="menu-icon">
```
Comment (marynakorol) [inline AltexSoft lab 1/index.html]: better to use CSS to do the hamburger menu and transform elements to cross after click

File: AltexSoft lab 1/scss/_media-query.scss [hunk 32]
```diff
@@ -0,0 +1,337 @@
...
+              color: inherit;
+              margin-right: 50px;
+              padding-bottom: 7px;
+
+              &.active {
+                border-bottom: 1.5px solid $white-color;
+              }
+
+              &:hover {
+                color: $color-for-links-and-buttons;
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/_media-query.scss]: simplify var name

File: AltexSoft lab 1/scss/_media-query.scss [hunk 33]
```diff
@@ -0,0 +1,337 @@
...
+      .menu-btn {
+        .menu-icon {
+          display: none;
+        }
+      }
+    }
+  }
+
+  .navbar {
+    height: 80px;
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/_media-query.scss]: do not hardcode height
image

File: AltexSoft lab 1/scss/_media-query.scss [hunk 34]
```diff
@@ -0,0 +1,337 @@
+@import "variables";
+
+@media only screen and (min-width: 1440px) {
+  .header {
+    height: 500px;
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/_media-query.scss]: do not hardcode height
image

File: AltexSoft lab 1/scss/_media-query.scss [hunk 35]
```diff
@@ -0,0 +1,337 @@
...
+
+    .header-title {
+      padding-top: 182px;
+      font-size: 110px;
+      line-height: 130px;
+      width: 50%;
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/_media-query.scss:9]: uppercase missed

File: AltexSoft lab 1/scss/_media-query.scss [hunk 36]
```diff
@@ -0,0 +1,337 @@
...
+                font-size: 13px;
+              }
+            }
+          }
+        }
+
+        .accordion-btn {
+          height: 148.5px;
+
+          &.active {
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/_media-query.scss:173]: doesn't match the design 
image
if text content is small it is justified at the bottom

File: AltexSoft lab 1/scss/_media-query.scss [hunk 37]
```diff
@@ -0,0 +1,337 @@
...
+            .accordion-title {
+              font-size: 42px;
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/_media-query.scss]: use var

File: AltexSoft lab 1/scss/_media-query.scss [hunk 38]
```diff
@@ -0,0 +1,337 @@
...
+              line-height: 54px;
+              max-width: 359px;
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/_media-query.scss]: do not use magic numbers

File: AltexSoft lab 1/scss/_media-query.scss [hunk 39]
```diff
@@ -0,0 +1,337 @@
...
+        }
+      }
+    }
+
+    .skout-glossary {
+      .skout-glossary-img-wrapper {
+        padding: 0 135px;
+        .skout-glossary-img {
+          height: 41px;
+          margin: 19.5px 0 50px 0;
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/_media-query.scss]: magic numbers, do not use decimal points for px

File: AltexSoft lab 1/scss/_media-query.scss [hunk 40]
```diff
@@ -0,0 +1,337 @@
...
+          font-size: 40px;
+        }
+      }
+    }
+
+    .skout-glossary {
+      .skout-glossary-img-wrapper {
+        padding: 0 135px;
+        .skout-glossary-img {
+          height: 41px;
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/_media-query.scss]: move to ::before

File: AltexSoft lab 1/scss/_media-query.scss [hunk 41]
```diff
@@ -0,0 +1,337 @@
...
+          margin: 19.5px 0 50px 0;
+        }
+      }
+
+      .skout-glossary-slider {
+        padding: 60px 0;
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/_media-query.scss]: image
padding is different for every new content, arrows should be like on design

Comment (marynakorol) [reply AltexSoft lab 1/scss/_media-query.scss]: image

File: AltexSoft lab 1/scss/_media-query.scss [hunk 42]
```diff
@@ -0,0 +1,337 @@
...
+              .skout-glossary-slider-menu-line {
+                width: 105px;
+              }
+            }
+          }
+        }
+      }
+    }
+
+    .skout-faqs {
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/_media-query.scss:270]: image

Comment (marynakorol) [reply AltexSoft lab 1/scss/_media-query.scss:270]: doesn't match the design

File: AltexSoft lab 1/scss/_media-query.scss [hunk 43]
```diff
@@ -0,0 +1,337 @@
...
+      padding: 0 135px;
+
+      .skout-faqs-more-questions-btn {
+        margin: 50px auto 102px auto;
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/_media-query.scss]: change 102 to reusable size

File: AltexSoft lab 1/scss/_media-query.scss [hunk 44]
```diff
@@ -0,0 +1,337 @@
...
+      }
+
+      .skout-faqs-img {
+        height: 32px;
+        margin: 19.5px 0 100px 0;
+      }
+
+      .skout-faqs-accordion {
+        .skout-faqs-accordion-btn {
+          height: 196px;
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/_media-query.scss]: !

File: AltexSoft lab 1/scss/_media-query.scss [hunk 45]
```diff
@@ -0,0 +1,337 @@
...
+            margin-left: 100px;
+          }
+        }
+
+        .skout-faqs-answer {
+          .skout-faqs-answer-wrapper {
+            .skout-faqs-a-letter {
+              font-size: 110px;
+              line-height: 130px;
+            }
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/_media-query.scss:299]: font-size: 110px;    
line-height: 130px;

code duplicate 
GLOBAL: keep such values in classes or mixins

File: AltexSoft lab 1/scss/_media-query.scss [hunk 46]
```diff
@@ -0,0 +1,337 @@
...
+
+            .skout-faqs-answer-text {
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/_media-query.scss:301]: image
fix indents according to design

File: AltexSoft lab 1/scss/_media-query.scss [hunk 47]
```diff
@@ -0,0 +1,337 @@
...
+              font-size: 16px;
+              max-width: 670px;
+            }
+          }
+        }
+      }
+    }
+  }
+
+  .footer {
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/_media-query.scss:313]: image
limit first block max-width, be sure all content is displaying properly

Comment (marynakorol) [reply AltexSoft lab 1/scss/_media-query.scss:313]: tel must be a link and have its hover effect

File: AltexSoft lab 1/scss/_media-query.scss [hunk 48]
```diff
@@ -0,0 +1,337 @@
...
+              margin: 0 0 0 auto;
+              font-size: 16px;
+              line-height: 24px;
+            }
+          }
+        }
+      }
+
+      .footer-delimiter {
+        margin: 0 0 28px 0;
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/_media-query.scss]: use margin-bottom here

File: AltexSoft lab 1/scss/_media-query.scss [hunk 49]
```diff
@@ -0,0 +1,337 @@
...
+        justify-content: space-between;
+        align-items: center;
+
+        .footer-social-block {
+          .icon {
+            margin-right: 53px;
+          }
+        }
+
+        .footer-logo-wrapper {
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/_media-query.scss:378]: logo should be a link

File: AltexSoft lab 1/scss/_media-query.scss [hunk 50]
```diff
@@ -0,0 +1,337 @@
...
+          margin: 0;
+          .footer-logo {
+            height: 58px;
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/_media-query.scss]: :-( height

File: AltexSoft lab 1/scss/_variables.scss [hunk 51]
```diff
@@ -0,0 +1,5 @@
+$white-color: #fff;
+$color-for-links-and-buttons: #22BF95;
+$main-gray-color: #7F7F7F;
+$secondary-gray-color: #E5E5E5;
+$delimiter-color: #B2B2B2;
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/_variables.scss]: use consistent register for hex values #fff -> #FFFFFF

File: AltexSoft lab 1/scss/_media-query.scss [hunk 52]
```diff
@@ -0,0 +1,337 @@
...
+    .skout-glossary {
+      .skout-glossary-img-wrapper {
+        padding: 0 135px;
+        .skout-glossary-img {
+          height: 41px;
+          margin: 19.5px 0 50px 0;
+        }
+      }
+
+      .skout-glossary-slider {
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/_media-query.scss]: this block should have full width bg

File: AltexSoft lab 1/scss/style.scss [hunk 53]
```diff
@@ -0,0 +1,726 @@
+@import "variables";
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.header {
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/style.scss:14]: image
fix all red points: indents, font-sizes, collapsed elements, elements that do not wrap

File: AltexSoft lab 1/scss/style.scss [hunk 54]
```diff
@@ -0,0 +1,726 @@
...
+
+  .skout-tips {
+    padding: 0 20px;
+    margin: 0 auto;
+    .skout-tips-img {
+      height: 18px;
+      margin: 10px 0 24px 0;
+    }
+
+    .accordion-wrapper {
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/style.scss:256]: indent missed

File: AltexSoft lab 1/js/pagination.js (hunk 26 above)
Comment (marynakorol) [reply AltexSoft lab 1/js/pagination.js]: re-init slider on resize

**Review by marynakorol — CHANGES_REQUESTED** [review]

File: AltexSoft lab 1/scss/_media-query.scss (hunk 42 above)
Comment (FlameNV) [reply AltexSoft lab 1/scss/_media-query.scss:270]: Why? A top red rectangle has a height of 160px, like in Figma. A bottom red rectangle has a height of 100px, like in Figma.

File: AltexSoft lab 1/scss/_media-query.scss (hunk 46 above)
Comment (FlameNV) [reply AltexSoft lab 1/scss/_media-query.scss:301]: Fixed top red rectangle. The bottom red rectangle like in Figma.

File: AltexSoft lab 1/index.html (hunk 8 above)
Comment (marynakorol) [reply AltexSoft lab 1/index.html]: use span here instead of p

File: AltexSoft lab 1/index.html (hunk 16 above)
Comment (marynakorol) [reply AltexSoft lab 1/index.html]: please remove all spaces inside href

File: AltexSoft lab 1/index.html (hunk 8 above)
Comment (FlameNV) [reply AltexSoft lab 1/index.html]: ?? In the second commit, I fixed it

File: AltexSoft lab 1/scss/_media-query.scss [hunk 55]
```diff
@@ -0,0 +1,391 @@
...
+      .menu-btn {
+        .menu-icon {
+          display: none;
+        }
+      }
+    }
+  }
+
+  .navbar {
+    margin-bottom: 98px;
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/_media-query.scss]: round this value

File: AltexSoft lab 1/scss/_media-query.scss [hunk 56]
```diff
@@ -0,0 +1,391 @@
...
+    border-top: 5px solid $color-clicable-el;
+    top: 100px;
+    padding: 30px 0;
+
+    .navbar-link {
+      padding: 0 13%;
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/_media-query.scss]: 13% is incorrect value, use px

File: AltexSoft lab 1/scss/_media-query.scss [hunk 57]
```diff
@@ -0,0 +1,391 @@
...
+      &.margin {
+        margin-top: 160px;
+      }
+    }
+
+    .skout-tips {
+      padding: 0 135px;
+
+      .skout-tips-img {
+        width: 247px;
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/_media-query.scss]: needs to be rounded to avoid magic numbers

File: AltexSoft lab 1/scss/_media-query.scss [hunk 58]
```diff
@@ -0,0 +1,391 @@
...
+
+            .accordion-img {
+              width: $size-accordion-image-desktop;
+              height: $size-accordion-image-desktop;
+            }
+          }
+
+          .accordion-content {
+            display: inline-block;
+            width: 464px;
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/_media-query.scss]: limit the width via indents, not the width property

File: AltexSoft lab 1/scss/style.scss [hunk 59]
```diff
@@ -0,0 +1,782 @@
...
+          color: $color-white;
+          transition: 0.7s;
+        }
+      }
+
+      .menu-get-skout-btn {
+        display: block;
+        padding: 7px 54px;
+        margin-top: 205px;
+      }
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/style.scss:171]: image
the number of links can be different and these buttons will be invisible due to hardcoding of this indentation

File: AltexSoft lab 1/scss/style.scss [hunk 60]
```diff
@@ -0,0 +1,782 @@
...
+        display: block;
+        padding: 7px 23px;
+        margin-top: 16px;
+      }
+    }
+  }
+
+  .header-title {
+    display: block;
+    width: 38%;
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/style.scss]: inappropriate width value

File: AltexSoft lab 1/scss/style.scss [hunk 61]
```diff
@@ -0,0 +1,782 @@
...
+  }
+}
+
+.navbar {
+  position: sticky;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  max-width: 770px;
+  width: 89.4%;
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/style.scss]: the same

File: AltexSoft lab 1/scss/style.scss [hunk 62]
```diff
@@ -0,0 +1,782 @@
...
+  }
+}
+
+.main-section {
+  min-width: 335px;
+  margin: 0 auto;
+
+  .section-delimiter {
+    height: 1px;
+    width: 89.4%;
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/style.scss]: !

File: AltexSoft lab 1/scss/style.scss [hunk 63]
```diff
@@ -0,0 +1,782 @@
...
+      .accordion-btn {
+        @include letters-m;
+
+        display: flex;
+        align-items: center;
+        width: 100%;
+        border: none;
+        outline: none;
+        cursor: pointer;
+        transition: 0.4s;
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/style.scss]: better to keep animation transition duration consistent through the site

File: AltexSoft lab 1/scss/style.scss [hunk 64]
```diff
@@ -0,0 +1,782 @@
...
+      &:hover {
+        color: $color-primary-dark;
+        transition: 0.7s;
+      }
+    }
+  }
+}
+
+.footer {
+  background-color: #f2f2f2;
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/style.scss]: use var

File: AltexSoft lab 1/scss/style.scss [hunk 65]
```diff
@@ -0,0 +1,782 @@
...
+
+      &:hover {
+        transition: 0.7s;
+        color: $color-clicable-el;
+      }
+    }
+
+    .footer-info-block {
+      .footer-nav-block {
+        padding: 24px 0 20px 23px;
```
Comment (marynakorol) [inline AltexSoft lab 1/scss/style.scss]: use paired indents

**Review by marynakorol — CHANGES_REQUESTED** [review]
needs just some changes

**Review by marynakorol — APPROVED** [review]

**Comment by marynakorol** [comment]
you can use an easier way for padding and margin  properties in future:), e.g.:
 padding: 24px 0 20px 0; --> should be padding: 24px 0 20px; 
 padding: 24px 0 24px 0; --> padding: 24px 0;

**Pull request merged**